docs/database-schema.md

```diff
@@ -0,0 +1,102 @@
+# Interpolation Database Schema
+
+The Pelias Interpolation engine currently uses two [SQLite](https://sqlite.org) databases for
+storing street and address information. This document explains the most important tables in these
+databases and the meanings of the various fields.
+
+## Street database
+
+The street database contains several tables with data regarding streets.
+
+### `polyline` table
+
+| field | type | description |
+| --- | --- | --- |
+| id | `INTEGER` | the primary key for a given street |
+| line | `TEXT` | polyline encoded linestring |
+
+The polylines table contains a unique identifier for each street, as well as its geometry as a
+[polylines encoded string](https://developers.google.com/maps/documentation/utilities/polylinealgorithm) with precision `6`.
+
+Tools for working with polyline-encoded strings include:
+- [@mapbox/polyline](https://github.com/mapbox/polyline) Node.js package
+- [polyline](https://pypi.org/project/polyline/) Python package.
+
+
+### `names` table
+
+The names table contains information on names for a given street. Streets can have multiple names,
+so there may be multiple rows in the `names` table for a given row in the `polyline` table.
+
+| field | type | description |
+| --- | --- | --- |
+| rowid | `INTEGER` | the primary key column for this table, not generally referenced |
+| id | `INTEGER` | a foreign key to the corresponding row in the `polyline` table |
+| name | `TEXT` | the name of the street |
+
+## Address database
+
+The address database contains a single table with information on notable points associated with
+streets. These points might represent an exact address, the start or end of an address range, or
+merely a point along a curved street.
+
+### `address` table
+
+| field | type | description |
+| --- | --- | --- |
+| rowid | `INTEGER` | the primary key column for this table, not generally referenced |
+| id | `INTEGER` | a foreign key to the corresponding row in the `polyline` table |
+| source | `TEXT` | the datasource for this point |
+| source_id | `TEXT` | a unique identifier for this point from the original datasource (if applicable) |
+| housenumber | `REAL` | an exact or interpolated housenumber for this point |
+| lat | `REAL` | the latitude of this address point |
+| lon | `REAL` | the longitude of this address point |
+| parity | `TEXT` | Either `L` or `R` if this is an exact address, denoting the side of the street the address is on  |
+| proj_lat | `REAL` | The projected point along the related street corresponding to this address |
```

Comment:
Can maybe clarify this further with wording such as 'closest point on the street (eg where the letterbox is located)